js/content.js

```diff
@@ -513,6 +491,58 @@ function makeContent()
 					"</table>"+
 				"</fieldset>"+
 			"</div>"+
+			"<div id=\"st_fmt\" class=\"stg_con\">"+
+				"<fieldset>"+
+					"<legend>"+theUILang.Format+"</legend>"+
+					"<div class=\"op50l\">"+
+						"<label for=\"webui.dateformat\">"+theUILang.DateFormat+":</label>&nbsp;"+
+						"<select id=\"webui.dateformat\">"+
+							"<option value='0'>31.12.2011</option>"+
+							"<option value='1'>2011-12-31</option>"+
+							"<option value='2'>12/31/2011</option>"+
+						"</select>"+
+					"</div>"+
+					"<div class=\"op100l\"><input type=\"checkbox\" id=\"webui.show_statelabelsize\"/>"+
+						"<label for=\"webui.show_statelabelsize\" id=\"lbl_webui.show_statelabelsize\" >"+theUILang.showStateLabelSize+"</label>"+
+					"</div>"+
+					"<div class=\"op100l\"><input type=\"checkbox\" id=\"webui.show_labelsize\"/>"+
+						"<label for=\"webui.show_labelsize\" id=\"lbl_webui.show_labelsize\" >"+theUILang.showLabelSize+"</label>"+
+					"</div>"+
+					"<div class=\"op100l\"><input type=\"checkbox\" id=\"webui.show_searchlabelsize\"/>"+
+						"<label for=\"webui.show_searchlabelsize\" id=\"lbl_webui.show_searchlabelsize\" >"+theUILang.showSearchLabelSize+"</label>"+
+					"</div>"+
+					"<div class=\"op100l\"><input type=\"checkbox\" id=\"webui.show_label_path_tree\"/>"+
+						"<label for=\"webui.show_label_path_tree\" id=\"lbl_webui.show_label_path_tree\" >"+theUILang.showCustomLabelTree+"</label>"+
+					"</div>"+
+					"<div class=\"op100l\"><input type=\"checkbox\" id=\"webui.show_empty_path_labels\"/>"+
+						"<label for=\"webui.show_empty_path_labels\" id=\"lbl_webui.show_empty_path_labels\" >"+theUILang.showEmptyPathLabel+"</label>"+
+					"</div>"+
+				"</fieldset>"+
+				"<fieldset>"+
+					"<legend>"+theUILang.DecimalPlacesSizes+"</legend>"+
+					$('<table>').append(
+						$('<tr>').append(...[ '', 'Default', 'KB', 'MB', 'GB', 'TB', 'PB'].map((unit) =>
+							$('<th>').text(unit !== '' ? theUILang[unit] : ''))
+						),...Object.entries({
+							catlist: theUILang.CatListSizeDecimalPlaces,
```

Comment:
What we will have here for non-english languages?

Reply:
That appears to be indexing theUILang where is retrieves the unit translation. The values are being mapped to the variable "unit" then the text is being set as theUILang[unit]. I didn't test this, but it should theoretically work.

Reply:
I told about different thing. @TrimmingFool added keys ```*ListSizeDecimalPlaces``` to the English locale only. They undefined in the other locales.

Reply:
It would have shown: `undefined`
I updated it now

Reply:
Awesome, thank you for implementing this feature.